src/sharedTypes.ts

```diff
@@ -0,0 +1,3 @@
+//Todo better name
+export type HostingContextType = "home"|"custom"|"none"
```

Comment:
are we expecting more types here ?
do these values ("home"|"custom"|"none") correspond to anything in Dynamo ?

Reply:
yes, I can make it clearer with a comment.
"home" is a homeworkspace, custom is customNodeWorkspace. I don't think we'd expect anything else here - and currently only "home" does anything.

Reply:
Why not an Enum ?
It would be easier to compare, I think.
HostingContextType.Home == state.hostContext;

Reply:
@pinzart90 - I figured this out, using string enums in newer versions of typescript works well and is easy to use from js as well.